src/components/app/content/index.js

```diff
@@ -44,25 +59,14 @@ function Content() {
         }
     }
 
-    const [addCardModalVisible, setAddCardModalVisible] = useState(false)
-    const toggleAddCardModal = () => setAddCardModalVisible(prevState => !prevState)
-
-    function deleteCards() {
-        setCards(cards.filter(card => !checkedCardIds.includes(card.id)));
-        setCheckedCardIds([]);
-    }
-
-    function saveCardData(cardId, tempState) {
-        let card = cards.find(card => card.id === cardId);
-        card.header = tempState.header;
-        card.body = tempState.body;
-        setCards(cards);
-    }
+    const [addCardModalVisible, setAddCardModalVisible] = useState(false);
+    const toggleAddCardModal = () => setAddCardModalVisible(prevState => !prevState);
 
-    function addCard(cardData) {
-        const generatedId = uuidv4()
-        setCards([...cards, {id: generatedId, ...cardData}])
-    }
+    const [firstTimeLoadedIds, setFirstTimeLoadedIds] = useState([...cards.map(card => card.id)]);
+    const clearFirstTimeLoadedIds = () => setFirstTimeLoadedIds([]);
+    useEffect(() => {
+        setTimeout(() => setFirstTimeLoadedIds([]), 2000)
+    }, [cards])
```

Comment:
Looks good. But the timer should be placed in the `WithLoadingDelay` HOC-component, since specifically this component should add such **'2s load delay and spinner display'** functionality to your `Card` component (and any other if wrap it).